docs/web-ui.md

```diff
@@ -426,9 +426,9 @@ queries. Currently, it contains the following metrics.
 * **Batch Duration.** The process duration of each batch. 
 * **Operation Duration.** The amount of time taken to perform various operations in milliseconds.
 The tracked operations are listed as follows.
-    * addBatch: Adds result data of the current batch to the sink.
-    * getBatch: Gets a new batch of data to process.
-    * latestOffset: Gets the latest offsets for sources. 
+    * addBatch: Time taken to read the micro-batch's input data from the sources, process it, and write the batch's output to the sink. This should take the bulk of the micro-batch's time.
+    * getBatch: Time taken to prepare the logical query to read the input of the current micro-batch from the sources.
+    * latestOffset: Time taken to query the sources whether they have new input data.
```

Comment:
This seems not be complete - it retrieves the next "offsets" to process in the micro-batch which will be committed into offset log (`walCommit`), not just querying to see whether a new input data is available or not.

Reply:
+1

Reply:
Thanks, done in cb82167.